source/_integrations/light.template.markdown

```diff
@@ -99,6 +115,26 @@ light:
         required: false
         type: template
         default: optimistic
+      supports_transition_template:
+        description: Defines a template to get if light supports transition.
```

Comment:
Explain also that:
- What the template should return (True/False?)
- That a transition in a turn on or turn off call will be passed as a named parameter "transition" to either of the scripts.

Reply:
done